charts/rancher-istio/1.5.700/templates/crd-certmanager-10.yaml

```diff
@@ -9,14 +9,14 @@ metadata:
     heritage: Tiller
     release: istio
   annotations:
-    helm.sh/resource-policy: keep
-    helm.sh/hook: crd-install
+    "helm.sh/resource-policy": keep
+    helm.sh/hook: crd-install,pre-upgrade
```

Comment:
Note that I added pre-upgrade to all CRDs, previously they only used crd-install

Reply:
Is this for handling the case that Helm doesn't upgrade CRD?

Reply:
That case, and also when a new CRD is added the upgrade doesn't install it because `crd-install` hook is not run on upgrade.

Reply:
Got it.